config/environments/development.rb

```diff
@@ -36,6 +36,20 @@
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
+  #Paperclip
+  Paperclip.options[:command_path] = ENV["IMAGE_MAGICK_FOLDER"]
+
+  config.paperclip_defaults = {
+    :storage => :s3,
+    :s3_credentials => {
+      :bucket => ENV['S3_BUCKET_NAME'],
+      :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
+      :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+    },
+    :url => ':s3_domain_url',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.